Makefile.am

```diff
@@ -22,7 +22,7 @@ LIBS_ALL                += -lzmq
 
 ## Add Architecture-specific stuff
 if DARKSTAR_ARCH_LINUX
-LIBS_ALL                += -ldl
+LIBS_ALL                += -ldl -lpthread
```

Comment:
Is there a reason this was not forced on line 12?

Reply:
The bug was only reported on one individual flavour of linux and I didn't want to force this anywhere that it isn't 100% required. If I put it on line 12 it would also affect someone trying to use the makefile on Windows (however rare that may be)